open-source-deps.md

```diff
@@ -1,9 +1,9 @@
 Open-Source
 ===========
 
-Kubecost has an open core business model. 
+Kubecost has open source at the center of our business model. 
```

Comment:
open-core is a specific business model. I would at least call that out?

like:
```
Kubecost has open source at the center of our business model (open-core). 
```

But I agree that it was previously vague and assumed the reader understood open-core.

Reply:
Agreed, added back open-core.